index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="pt-br">
```

Comment:
Pela documentação, o br é uppercase
```suggestion
<html lang="pt-BR">
```

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang

Reply:
feito, mestre